cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java

```diff
@@ -306,6 +307,8 @@ public static void importCsvFromFile(
     } catch (StatementExecutionException e) {
       System.out.println(
           "Encounter an error when executing the statement, because " + e.getMessage());
+    } catch (TTransportException e) {
+      System.out.println("Encounter an error when connecting to server, because " + e.getMessage());
```

Comment:
I think slf4j has been imported, and it is recommended to use logger to output logs.

Reply:
I just follow the previous code style, it should be modified together.